mypy.ini

```diff
@@ -38,6 +38,10 @@ files =
     tools/generate_torch_version.py,
     tools/stats_utils/*.py
 
+#
+# `exclude` is a regex, not a list of paths like `files` (sigh)
+#
+exclude = torch/include/|torch/csrc/|torch/distributed/elastic/agent/server/api.py
```

Comment:
Do you know why it is looking into `torch/include` folder to begin with?

Reply:
Mypy got more agressive with searching in `0.800`, including paths that are not part of the Python package.  It only excludes `__pycache__`, `site-packages`, hidden directories, and maybe a few other directories like that. That of course led to a lot of breakage, so the `exclude` filter was added in `0.812`. 

 It also got slower because it's searching more. There may be something to gain later from adding more exclude paths, but I don't think making a giant one-line regex is helpful at this point.